test/onnx/test_models.py

```diff
@@ -143,13 +143,11 @@ def test_resnet(self):
         x = Variable(torch.randn(BATCH_SIZE, 3, 224, 224).fill_(1.0))
         self.exportTest(toC(resnet50()), toC(x), atol=1e-6)
 
-    @unittest.skip(
-        "This test has been flaky on trunk and PRs. See https://github.com/pytorch/pytorch/issues/79540"
-    )
     @skipScriptTest(min_opset_version=15)  # None type in outputs
+    # This test is numerically unstable. Sporadic single element mismatch occurs occasionally.
```

Comment:
Trying to understand context: Is single element mismatch not a bug?

Reply:
Good question. I think the backend folks would have a better say. It's definitely in a gray area. But I wouldn't want to disable the entire test until everything is fixed either. 
rtol/atol won't work in this case. It's a kind of "spike" mismatch, very large diff in value.

Reply:
Gotcha!